src/Http/Batch.php

```diff
@@ -207,7 +207,7 @@ private function parseRawHeaders($rawHeaders)
         list($header, $value) = explode(': ', $headerLine, 2);
         $header = strtolower($header);
         if (isset($headers[$header])) {
-          $headers[$header] .= "\n" . $value;
+          $headers[$header] .= ", " . $value;
```

Comment:
Shouldn't this be without an extra space?

```suggestion
          $headers[$header] .= "," . $value;
```

ref: https://github.com/guzzle/psr7/issues/494#issuecomment-1075520105 makes no mention of the space.

Reply:
I'm following the documentation provided by Mozilla. In the example it is displaying the space.

```
Vary: *
Vary: <header-name>, <header-name>, ...
```

Source: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Vary

Reply:
makes sense!